tests/js/smartstream_map/map.js

```diff
@@ -16,14 +16,18 @@ export const setup = async () => {
     } catch (e) {
       try {
         await admin.deleteTopic(topic);
-      } catch (e) {}
+      } catch (e) { }
       console.error(`${e.message}`);
     }
   }
+  let utf8Encode = new TextEncoder();
+  let wasm = utf8Encode.encode(mapCode)
```

Comment:
Huh. Interesting. It's unclear why this is needed now but wasn't needed before.

Reply:
I think that the transformation from js string to  Vec<u8> does not work the same way that in rust, and therefore we have some issues with this, I will test using String instead of Vec<u8> in the create_smartmodule function to see if that removes the issue

Reply:
Yeah, I think I ran into something similar a while back. If it's better as a `Vec<u8>`, that's fine. Most of the js folks I know don't like bytes is all.